jac-splice-orc/setup.py

```diff
@@ -21,11 +29,17 @@
         "requests",
         "python-dotenv",
         "numpy",
+        "jaclang",
```

Comment:
We should add version requirements range for these packages.  Take a look at the setup.py in jac-cloud as reference.